README.md

```diff
@@ -1,16 +1,93 @@
-# bars_frontend
+# EMPOWER
 
-First frontend of our app.
+An app that visualizes the effects of changing inputs on the results of given logistic regression outputs.
```

Comment:
@justus-hildebrand

Reply:
```suggestion
Empower is an app that shows the relations between changes in feature vectors and the resulting changes in predictions made by pre-trained logistic regressors. It does this by providing users with an input interface to make changes to an instance's features and on the fly presenting the resulting changes of the model's prediction results.
```

Reply:
That is sooo complicated..It might be accurate, bu probably good as second sentence? So like "visualization" first and then "To be more exact <your part>".